app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -13,14 +13,22 @@ def github
   end
 
   def failure
-    Rails.logger.error "Omniauth failure",
-                       omniauth_failure: {
-                         error: request.env["omniauth.error"]&.inspect,
-                         error_type: request.env["omniauth.error.type"].to_s,
-                         auth: request.env["omniauth.auth"],
-                         provider: request.env["omniauth.strategy"].to_s,
-                         cookie: request.env["rack.request.cookie_hash"]
-                       }
+    error = request.env["omniauth.error"]
+
+    DatadogStatsClient.increment(
+      "omniauth.failure",
+      tags: [
+        "class:#{error}",
```

Comment:
Are we sure this shouldn't be `error.class` or something?

Reply:
`error.to_s` is the same as `error.class.name` in this case so I kept it like that 🤷‍♂

Reply:
No worries @rhymes, just double checking.